ding/data/tests/test_shm_buffer.py

```diff
@@ -18,3 +20,11 @@ def test_shm_buffer():
     proc = mp.Process(target=subprocess, args=[shm_buf])
     proc.start()
     proc.join()
+
+
+@pytest.mark.benchmark
```

Comment:
maybe `cudatest`?

Reply:
Done, and also add "cudatest" marker in test_torchrpc_cuda()